src/clusterfuzz/_internal/metrics/monitoring_metrics.py

```diff
@@ -273,6 +273,17 @@
         monitor.StringField('status'),
     ])
 
+UNTRIAGED_TESTCASE_AGE = monitor.CumulativeDistributionMetric(
+    'issues/untriaged_testcase_age',
+    description='Age of testcases that were not yet triaged '
+    '(have not yet completed analyze, progression,'
```

Comment:
Progression does not seem to be part of critical tasks, you probably meant regression here.